security/fma-superchainerc20.md

```diff
@@ -0,0 +1,101 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## **Table of Contents** *generated with [DocToc](https://github.com/thlorenz/doctoc)*  
+
+- [**SuperchainERC20 standard-only FMA (Failure Modes and Recovery Path Analysis)**](#superchainerc20-standard-only-fma-failure-modes-and-recovery-path-analysis)
+- [Introduction](#introduction)
+  - [Security Considerations](#security-considerations)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+  - [FM1: Unauthorized Access to `crosschainMint` & `crosschainBurn` Functions](#fm1-unauthorized-access-to-crosschainmint--crosschainburn-functions)
+  - [FM2: Token contract not Deployed in Destination Chain, but Able to Deploy](#fm2-token-contract-not-deployed-in-destination-chain-but-able-to-deploy)
+  - [FM3: Token Deployer is Lost, or Unable to Deploy to Expected Address](#fm3-token-deployer-is-lost-or-unable-to-deploy-to-expected-address)
+  - [FM4: Compromised Deployment Method](#fm4-compromised-deployment-method)
+- [Action Items](#action-items)
+- [Audit Requirements](#audit-requirements)
+
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## **SuperchainERC20 standard-only FMA (Failure Modes and Recovery Path Analysis)**
+
+| Author | Ng, Joxes, Particle, Gotzen |
+| --- | --- |
+| Created at | 2024-10-02 |
+| Needs Approval From | Mark Tyneway, Matt Solomon |
```

Comment:
Would be good to also get a quick review from someone on interop who is familiar with this for any big issues, @tynes could be good but could also be someone from wonderland

Reply:
Wonderland has reviewed internally, so resolving